ansible/playbooks/roles/opensearch/tasks/generate-certs.yml

```diff
@@ -5,40 +5,37 @@
   file:
     state: directory
     path: "{{ certificates.dirs.ca_key }}"
-    owner: root
-    group: elasticsearch
-    mode: u=rwx,g=rx,o=  # elasticsearch.service requires 'rx' for group
+    owner: "{{ specification.opensearch_os_user }}"
+    group: "{{ specification.opensearch_os_group }}"
+    mode: u=rwx,g=rwx,o=
 
 # csr files are kept only for idempotency
 - name: Create directory for CSR files
   file:
     state: directory
     path: "{{ certificates.dirs.csr }}"
-    owner: root
-    group: elasticsearch
-    mode: u=rwx,g=rx,o=  # CSR file doesn't contain private key
```

Comment:
Up to you but additional space before inline comment increases readability.
Thus 2 is the default value for `min-spaces-from-content` setting in yamllint (https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments).
Personally I'm fine with both styles: single space = faster typing and double = better readability 😉

Reply:
I see. I'm using default formating from the YAML VC extension which corrected it automatically.
![image](https://user-images.githubusercontent.com/13080132/168693237-4a6ab8bc-2209-455b-b513-15af0832a5be.png)